Flat/interface/PandaAnalyzer.h

```diff
@@ -208,6 +217,8 @@ public :
     void CalcBJetSFs(BTagType bt, int flavor, double eta, double pt, 
                      double eff, double uncFactor, double &sf, double &sfUp, double &sfDown);
     void ComplicatedLeptons();
+    void ComplicatedPhotons();
+    bool PFChargedPhotonMatch(panda::Photon photon);
```

Comment:
Can this be passed by const reference instead of value? Just `panda::Photon` -> `const panda::Photon&`. This avoids copying the whole object.